source/fundamentals/indexes.txt

```diff
@@ -371,3 +384,53 @@ The following example creates a unique, descending index on the ``theaterId`` fi
 
 Refer to the :manual:`Unique Indexes page </core/index-unique>` in the MongoDB server manual for more information.
 .. driver-content-end
+
+Remove an Index
```

Comment:
This shows specifically how to remove a text index, right? A non-text index you might use the helper to build the name depending on the sort direction, e.g.:
collection.dropIndex(Index.ascending("imdb"))
If you meant it to apply to one or the other, I think it could be helpful to identify either in this title or the text that immediately follows it.

Reply:
I added an additional section called "index specification document" and made the current section called "name field"